### ISSUE TITLE
"dotnet graphql update" does not refresh the schema file

### ISSUE THREAD

**Issue by joumdane** [ChilliCream/graphql-platform#6140]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Product

Strawberry Shake

### Describe the bug

When you run "dotnet graphql update" as documented in [the strawberryshake tooling documentation](https://chillicream.com/docs/strawberryshake/v13/tooling), it does not download and update the schema file with the newest version of the GraphQL server.

### Steps to reproduce

1. Make a small update on a query or mutation in your GraphQL server.
2. Run your GraphQL server.
3. Open your client project in a terminal.
4. Execute "dot graphql update"
5. You will see the following two messages:
> Download schema started.
> Download schema completed in xxx ms.

But the schema is not updated.

### Additional Context?

We have the same behavior either the terminal is launched in Visual studio 2022, Rider, Visual Studio Code or using PowerShell.

As workaround, we use: "dotnet graphql download https://localhost/graphql".

### Version

13.0.5

**Comment by nih0n** [comment]
This is fixed in `13.1.0-preview.5` and in the latest `14.0.0` preview, the bug was due this if else

Here's the current version `13.0.5`

```csharp
if (!await DownloadSchemaAsync(context, uri, tempFile, cancellationToken)
    .ConfigureAwait(false))
{
    // if the schema download succeeded we will replace the old schema with the
    // new one.
    if (File.Exists(schemaFilePath))
    {
        File.Delete(schemaFilePath);
    }

    File.Move(tempFile, schemaFilePath);

    hasErrors = true;
}
```

Here's the `13.1.0-preview.5`

```csharp
if (!await DownloadSchemaAsync(context, uri, tempFile, cancellationToken)
    .ConfigureAwait(false))
{
    hasErrors = true;
}
else
{
    // if the schema download succeeded we will replace the old schema with the
    // new one.
    if (File.Exists(schemaFilePath))
    {
        File.Delete(schemaFilePath);
    }

    File.Move(tempFile, schemaFilePath);
}
```

@michaelstaib should I create a pull request for this ?

**Comment by joumdane** [comment]
@nih0n, thank you for your feedback and reactivity, the command line "dotnet graphql update" is working fine with Strawberry tools v. "13.1.0-preview.6".
